platform/default/src/mbgl/storage/mbtiles_file_source.cpp

```diff
@@ -42,7 +42,8 @@ using namespace rapidjson;
 
 class MBTilesFileSource::Impl {
 public:
-    explicit Impl(const ActorRef<Impl>&, const ResourceOptions& options): resourceOptions (options.clone()) {}
+    explicit Impl(const ActorRef<Impl>&, const ResourceOptions& resourceOpts, const ClientOptions& clientOpts)
```

Comment:
Use full names.

Reply:
Same for everywhere: I can't as members are used like that. I did not want to break the current coding style not prefixing member variables (which is otherwise not that common).

Reply:
Done. Using `_` as suffix as in other classes.

Reply:
Great